lib/PuppeteerSharp.Tests/Puppeteer/PuppeteerLaunchTests.cs

```diff
@@ -66,5 +63,19 @@ public PuppeteerLaunchTests()
             await page.CloseAsync();
             await browser.CloseAsync();
         }
+
+        [Fact]
+        public async Task ShouldRejectAllPromisesWhenBrowserIsClosed()
+        {
+            var browser = await PuppeteerSharp.Puppeteer.LaunchAsync(TestConstants.DefaultBrowserOptions(), TestConstants.ChromiumRevision);
+            var page = await browser.NewPageAsync();
+
+            var neverResolves = page.EvaluateHandle("() => new Promise(r => {})");
```

Comment:
I missed this guy on this list https://github.com/kblok/puppeteer-sharp/issues/76.

If this works that would be great because `EvaluateHandle` is working, and I wasn't expecting that till v1.0. But it is far from working, lets ignore this test and move it to that list, so we cover the `EvaluateHandle` as a whole.